services/postServices.js

```diff
@@ -0,0 +1,17 @@
+var Post = require('../models/post.js');
+var moment = require('moment');
+
+var postServices = {};
+
+postServices.createPost = function(newPost, cb) {
+	newPost.createdAt = moment().toDate();
+	Post.create(newPost, function(err, post) {
+		cb(err, post);
+	});
+};
+
+postServices.findPost = function(cb) {
+	Post.find().exec(function(err, posts) {
```

Comment:
'`findPost`' probably isn't the best name for this function, since it's not finding a particular post, it's finding all of the posts.